pkg/chartverifier/report/types.go

```diff
@@ -26,6 +26,7 @@ type ReportMetadata struct {
 type ToolMetadata struct {
 	Version                    string  `json:"verifier-version" yaml:"verifier-version"`
 	Profile                    Profile `json:"profile" yaml:"profile"`
+	ReportDigest               string  `json:"report-digest" yaml:"report-digest"`
```

Comment:
Even though other keys are using `kebab-case` names, `snakeCase` should be used instead.

Reply:
Updated.